<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>phonePage</class>
 <widget class="QWidget" name="phonePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>380</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Turn on bluetooth on your mobile phone and select the appropriate device:

You don't need to choose one, you can configure it afterwards in the settings.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>558</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="DeviceScanWidget" name="devicesView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceScanWidget</class>
   <extends>QListView</extends>
   <header>widget.DeviceWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
